### ISSUE TITLE
Feature: Documentation about Rust dependencies

### ISSUE THREAD

**Issue by firstunicorn** [airtai/faststream#1939]
**Is your feature request related to a problem? Please describe.**
Hello FastStream Team, first of all, thanks for your project and your work.

I am evaluating FastStream with PydanticV2 written in Rust for my project and wanted to confirm whether it includes any Rust extensions or dependencies. 

I don't know Rust much {my project on Python - Django DRF}, but as far as I understand with Rust, you have a few options: 1) installing Rust on OS level {not the best option in a containerized environment} 2) Build "on the fly" while Docker container created {minimal build with only necessary parts for current container/app} 3) Just "classical" Rust installation inside of the container (similar as on OS level, but inside of the container}. 4) Python app can include some Rust part as a module/dependency, "built-in" / "embedded".

I want to use option 2, but if Rust in FastStream is already used as option 4, then I don't need to install anything.

TL;DR I am concerned about the necessity of having Rust installed when deploying FastStream in containerized environments.

Could you please provide insights into whether FastStream leverages Rust internally?

Thanks.

**Describe the solution you'd like**
Adding more information about Rust dependencies in the documentation and quick start.

**Describe alternatives you've considered**
If Rust is not included, it would be nice to have that option or (which will solve everything else) have a Docker container option.

**Additional context**
I use Django DRF + React on the frontend with the latest Python and the latest Ubuntu. I use Podman for containerization (it has "backward" combability with Docker and uses the same config files). I plan to use FastStream probably with RabbitMQ as a messaging broker for emailing micro-service (registration, product-related notifications, etc.). I use Prometheus if it's even relevant info (probably will use FastStream also for monitoring-related tasks and/or infrastructure tasks, but not sure right now).

**Comment by Rusich90** [comment]
Hello! The FastStream library does not use Rust at all, so no Rust installation is required to work with it.

As for Pydantic v2, while it is partially written in Rust, you generally do not need to install Rust to use it.

Why?
Precompiled binaries: Pydantic v2 is distributed as prebuilt wheels via pip, which include the Rust components already compiled. On most systems, you can install it easily: pip install pydantic


When is Rust needed?
   - If you’re on an unsupported platform without prebuilt binaries.
   -  If you want to compile Pydantic from source.

For typical environments, neither FastStream nor Pydantic will require you to have Rust installed.

**Comment by kumaranvpl** [comment]
Hello @firstunicorn,

As @Rusich90 mentioned, FastStream library doesn't use Rust. It uses pydantic library. 

In your docker container, simply install faststream using pip and it shoudl work without any problem.

**Comment by kumaranvpl** [comment]
Closing this issue.
